source/fundamentals/monitoring.txt

```diff
@@ -0,0 +1,396 @@
+==========
+Monitoring
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecols
+
+Overview
+--------
+
+This guide shows you how to monitor topology events in a MongoDB instance,
+replica set, or sharded cluster. Topology events, also known as Server
+Discovery and Monitoring (SDAM) events, are created by the driver when there is
+a change in the state of the instance or cluster that you are connected to.
+For example, when a new connection is established or a new primary is elected,
+an event is created.
+
+Read this guide if you need to record topology changes in your application or
+want to explore the information provided in these events.
+
+Event Descriptions
+------------------
+
+The following table lists all the SDAM events that are made available to your
+application by the driver:
```

Comment:
This table feels very reference-y. I would suggest moving it down lower in the page, towards the very bottom, for this reason. It might be a good idea to give a little more introductory exposition (explain the high level of how you subscribe to these events programmatically, for instance) instead before the first long-form example.

Reply:
On second thought, I think it would fit perfectly after the first example but before "Example event documents"

Reply:
Good idea, I'll review and update.

Reply:
I wonder if I should just show only one generic example of subscribing to an event since they haven't been introduced in the "Event Subscription Example" section which would be first in order if I made the change. I'll ask the team for their opinions.

Reply:
Decided to produce a generic example to support the reordering.